### PR TITLE
Technical debt

### PR DESCRIPTION
**Pull request by Alladin9393**
## Change Description

- Added to `swapparams` table `eth_swap_contract_address`, `eth_return_chainid` parameters
- Move `out_swap_min_amount` to the chains table (each chain has its own commission to token swap out remchain)
- Replacement `setchainid`, `setswapfee` actions to the `setswapparam` action
- Added new test cases

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Review by Art17 — APPROVED** [review]

File: contracts/contracts/rem.swap/include/rem.swap/rem.swap.hpp [hunk 1]
```diff
@@ -116,46 +112,42 @@ namespace eosio {
...
        *
-       * @param quantity - the quantity of tokens to be deducted.
+       * @param in_swap_fee - fee for token swap in remchain,
+       * @param chain_id - the chain identifier of the remchain,
+       * @param eth_swap_contract_address - the address of the ethereum token swap contract,
+       * @param eth_return_chainid - the chain identifier of the ethereum network for out token swap.
        */
       [[eosio::action]]
-      void setswapfee(const int64_t &amount);
+      void setswapparam(const int64_t &in_swap_fee, const string &chain_id,
```
Comment (Art17) [inline contracts/contracts/rem.swap/include/rem.swap/rem.swap.hpp]: make in_swap_fee separate for every chain_id

**Review by Art17 — CHANGES_REQUESTED** [review]

File: contracts/contracts/rem.swap/include/rem.swap/rem.swap.hpp (hunk 1 above)
Comment (Alladin9393) [reply contracts/contracts/rem.swap/include/rem.swap/rem.swap.hpp]: @Art17, done.

**Comment by Alladin9393** [comment]
Build_and_test

**Comment by Alladin9393** [comment]
Build_and_test

**Review by Art17 — APPROVED** [review]

**Pull request merged**